lib/xcode/install.rb

```diff
@@ -450,7 +450,14 @@ def prereleases
 
     def verify_integrity(path)
       puts `/usr/sbin/spctl --assess --verbose=4 --type execute #{path}`
-      $?.exitstatus.zero?
+      $?.exitstatus.zero? && verify_cert(path)
+    end
+
+    def verify_cert(path)
+      cert_info = Fastlane::Actions::VerifyBuildAction.gather_cert_info(path)
+      apple_team_identifier_result = cert_info['team_identifier'] == '59GAB85EFG'
+      apple_authority_result = cert_info['authority'].include?('Apple Mac OS Application Signing')
```

Comment:
Team identifier and authority are constants.

I am concerned whether they can be hard-coded into these line.

Reply:
I tried to fix (https://github.com/KrauseFx/xcode-install/pull/312/commits/adfa4007f73bc87c7a39d3fcf5b9fb6718289c5c).